showcase/app/routes/application.ts

```diff
@@ -5,9 +5,10 @@
 
 import Route from '@ember/routing/route';
 import { service } from '@ember/service';
+import type IntlService from 'ember-intl/services/intl';
 
 export default class ApplicationRoute extends Route {
-  @service intl;
+  @service declare intl: IntlService;
```

Comment:
```suggestion
  @service declare readonly intl: IntlService;
```
[Question] In the converted utilities file we add `readonly` to the `router`. Could we do the same here for the `intl` service?